docs/_sass/base/_fonts.scss

```diff
@@ -24,7 +24,3 @@
 	font-style: normal;
 	src: url("../fonts/woff2/iosevka-bold.woff2") format('woff2'), url("../fonts/woff/iosevka-bold.woff") format('woff'), url("../fonts/ttf/iosevka-bold.ttf") format('truetype');
 }
-// Google Fonts
-// -----------------------------------------------
-@import url('https://fonts.googleapis.com/css?family=Montserrat:400,600,700|Source+Code+Pro&display=optional');
-@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,600,700&display=swap');
```

Comment:
Open Sans shouldn't be removed, as it's being used indeed, just Montserrat and Source Code Pro